flow/storage/elasticsearch/elasticsearch.go

```diff
@@ -67,6 +67,14 @@ const flowMapping = `
 			}
 		},
 		{
+			"rtt": {
```

Comment:
you need to bump the index version

Reply:
done, was an overlap from tcpmetics patch